test/Main.hs

```diff
@@ -0,0 +1,33 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Main where
+
+import Reflex.Dom
+
+import qualified Reflex.Dom as Dom
+
+import ReflexJsx (jsx)
+
+main :: IO ()
+main = return ()
+
+-------------------------------------------------------------------------------
+-- Some examples that should compile.
+-------------------------------------------------------------------------------
+
+-- These test basic html parsing
+html1, html2, html3, html4 :: (MonadWidget t m) => m ()
+html1 = [jsx|<div/>|]
+html2 = [jsx|<div></div>|]
+html3 = [jsx|<div>Hello</div>|]
+html4 = [jsx|<h1>My <span style="color:red">Important</span> Heading</h1>|]
+
+-- These test antiquotation (MonadWidget values + dynamic attributes)
+anti1, anti2, anti3, anti4, anti5 :: (MonadWidget t m) => m ()
+anti1 = [jsx|<div>{Dom.text "Hello"}</div>|]
```

Comment:
If you want, you could try adding a nested `<div>` with quasiquotes inside. This will be especially cool to test when we have values being returned from within the jsx block!